lms/static/js/edxnotes/plugins/accessibility.js

```diff
@@ -0,0 +1,245 @@
+
+            switch (keyCode) {
+                case KEY.TAB:
+                    tabControls = this.getViewerTabControls();
+                    if (event.shiftKey) { // Tabbing backwards
+                        if (target.is(listing)) {
+                            _.last(tabControls).focus();
+                        }
+                        else {
+                            this.focusOnPreviousTabControl(tabControls, target);
+                        }
+                    } else { // Tabbing forward
+                        if (target.is(listing)) {
+                            _.first(tabControls).focus();
+                        }
+                        else {
+                            this.focusOnNextTabControl(tabControls, target);
+                        }
+                    }
+                    event.preventDefault();
+                    event.stopPropagation();
+                    break;
+                case KEY.ENTER:
+                case KEY.SPACE:
+                    if (target.hasClass('annotator-close')) {
+                        this.annotator.viewer.hide();
+                        this.focusOnHighlightedText(event);
+                    }
+                    break;
+                case KEY.ESCAPE:
+                    this.annotator.viewer.hide();
+                    this.focusOnHighlightedText(event);
+                    break;
+            }
+        },
+
+        onEditorKeyDown: function (event) {
+            var KEY = $.ui.keyCode,
+                keyCode = event.keyCode,
+                target = $(event.target),
+                editor, form, editorControls, save, cancel,
+                tabControls;
+
+            editor = this.annotator.element.find('.annotator-editor');
+            form = editor.find('.annotator-widget');
+            editorControls = editor.find('.annotator-controls');
+            save  = editorControls.find('.annotator-save');
+            cancel = editorControls.find('.annotator-cancel');
+
+            switch (keyCode) {
+                case KEY.TAB:
+                    tabControls = this.getEditorTabControls();
+                    if (event.shiftKey) { // Tabbing backwards
+                        if (target.is(form)) {
+                            _.last(tabControls).focus();
+                        } else {
+                            this.focusOnPreviousTabControl(tabControls, target);
+                        }
+                    } else { // Tabbing forward
+                        if (target.is(form)) {
+                            _.first(tabControls).focus();
+                        } else {
+                            this.focusOnNextTabControl(tabControls, target);
+                        }
+                    }
+                    event.preventDefault();
+                    event.stopPropagation();
+                    break;
+                case KEY.ENTER:
+                    if (target.is(save) || event.metaKey || event.ctrlKey) {
```

Comment:
@polesye 
> why do we need these checks for event.metaKey and event.ctrlKey ?

Because COMMAND/CONTROL + ENTER is supposed to be a shortcut to save notes in the Editor. Pressing ENTER will just add a new line.